source/whats-new.txt

```diff
@@ -36,43 +36,118 @@ Upcoming Breaking Changes
 
 What's New in 4.11
 ------------------
 
+.. important:: Deprecation Notice
+
+   - The ``getStats()`` and ``isCapped()`` instance methods of the
+     ``DBCollection`` class are deprecated. The corresponding server
+     commands are deprecated in MongoDB v6.2 and later. Use the ``$collStats`` aggregation
+     pipeline stage to retrieve the information provided by these methods
+     instead. You can run the aggregation as shown in the following code example:
+
+     .. code-block:: java
+
+        collection.aggregate(Arrays.asList(
+                         new Document("$collStats",
+                             new Document("storageStats", new Document()))));
+
+     To determine whether the collection is a capped collection, access the value
+     of the ``storageStats.capped`` field returned by this example aggregation.
+
+     To learn more about the ``$collStats`` aggregation operator, see the
+     :manual:`$collStats (aggregation) </reference/operator/aggregation/collStats/>`
+     Server manual entry.
+
+.. fulfills JAVA-4845: (from Upcoming Breaking Changes) getstats and iscapped deprecated.
+
+   - The following network address-related methods are deprecated:
+
+      - ``getSocketAddress()`` and ``getSocketAddresses()`` methods from the `ServerAddress <{+api+}/apidocs/mongodb-driver-core/com/mongodb/ServerAddress.html>`__ class
+      - ``getUnixSocketAddress()`` from the `UnixServerAddress <{+api+}/apidocs/mongodb-driver-core/com/mongodb/UnixServerAddress.html>`__ class
+
+.. fulfills DOCSP-32942
+
+      - The following Stream API methods and classes are deprecated and are
+        scheduled for removal in v5.0:
+
+        - ``streamFactoryFactory()`` method from ``MongoClientSettings.Builder``
+        - ``getStreamFactoryFactory()`` method from ``MongoClientSettings``
+        - ``NettyStreamFactoryFactory`` class
+        - ``NettyStreamFactory`` class
+        - ``AsynchronousSocketChannelStreamFactory`` class
+        - ``AsynchronousSocketChannelStreamFactoryFactory`` class
+        - ``BufferProvider`` class
+        - ``SocketStreamFactory`` class
+        - ``Stream`` class
+        - ``StreamFactory`` class
+        - ``StreamFactoryFactory`` class
+        - ``TlsChannelStreamFactoryFactory`` class
+
+.. fulfills DOCSP-33033
+
 .. note:: Upgraded Netty Version
 
    The {+driver-short+} now tests with version 4.1.87.Final of the
    ``io.netty:netty-all`` package. This change affects you only if you use the
    ``SslContext`` class that Netty provides.
 
-.. New features of the 4.11 driver release include:
+New features of the 4.11 driver release include:
+
+- Support for connecting to MongoDB by using a SOCKS5 proxy. To learn more
+  see :ref:`java-connect-socks`. TODO (DOCSP-31694)
+- Added a class to manage Client-Side Operations Timeout (CSOT) options.
+  To learn more, see: TODO https://jira.mongodb.org/browse/DOCSP-31693
+- Added the ``getSplitEvent()`` method to the ``ChangeStreamDocument`` class
```

Comment:
Do we have a usage example in any language?

Reply:
Not that I am able to locate. I could only find a server example: https://www.mongodb.com/docs/manual/reference/operator/aggregation/changeStreamSplitLargeEvent/

Do you think we need to create a ticket to show an example?

Reply:
Not sure if the page was updated, but it looks like there's an example that can help us get set up to test it. We will address the content/code example separate from this one for https://jira.mongodb.org/browse/DOCSP-31907